src/vs/platform/window/common/window.ts

```diff
@@ -91,16 +92,29 @@ export interface IFileToOpen extends IBaseWindowOpenable {
 	readonly fileUri: URI;
 }
 
-export function isWorkspaceToOpen(uriToOpen: IWindowOpenable): uriToOpen is IWorkspaceToOpen {
-	return !!(uriToOpen as IWorkspaceToOpen).workspaceUri;
+/**
+ * Represents an empty workspace with a remote authority.
+ * The empty string as remote authority signals that a local window should be opened.
+ * IEmptyWorkspace is currently only used in the web embedder API.
+ */
+export interface IEmptyWorkspace extends IBaseWindowOpenable {
```

Comment:
I wonder about terminology: should this rather be `IEmptyWindow` instead of empty workspace? In theory, a multi-root workspace could have 0 folders and also be called an empty workspace.

Reply:
`IEmptyWindow` is ok for me. But there can be files opened in it, so it's not always an empty window

I'm fine either way.